dune-project

```diff
@@ -84,7 +84,11 @@ structured way."))
 (package
  (name coqide)
  (depends
+  (ocamlfind :build)
+  (conf-findutils :build)
+  (conf-adwaita-icon-theme (>= 1))
```

Comment:
I think you can just write `conf-adwaita-icon-theme` in this line, not a big deal anyways.